app/lib/registries/organisations_registry.rb

```diff
@@ -0,0 +1,39 @@
+module Registries
+  class OrganisationsRegistry
+    CACHE_KEY = "registries/organisations".freeze
```

Comment:
Could do `CACHE_KEY = "#{NAMESPACE}/organisations".freeze` for consistency.

Reply:
I started with this, but for some reason it couldn't resolve it when running the tests 🤔